tdp/core/variables/variables.py

```diff
@@ -143,17 +146,22 @@ def _flush_on_disk(self) -> None:
         # https://docs.python.org/3/library/os.html#os.fsync
         os.fsync(self._file_descriptor.fileno())
 
-    def close(self) -> None:
+    def _close(self) -> None:
```

Comment:
This is not a private method, if a user do not want to use `with`, he must use this `close` method like the `open` function in Python.

Reply:
Fixed!